modules/inflowwind/src/InflowWind_IO.f90

```diff
@@ -2924,4 +2925,79 @@ subroutine Grid3D_WriteHAWC(G3D, FileRootName, unit, ErrStat, ErrMsg)
 
 end subroutine Grid3D_WriteHAWC
 
+
+subroutine Grid3D_WriteVTKsliceXY(G3D, FileRootName, XYslice_height, unit, ErrStat, ErrMsg)
+
+   type(Grid3DFieldType), intent(in)   :: G3D            !< Parameters
+   character(*), intent(in)            :: FileRootName   !< RootName for output files
+   real(ReKi),    intent(in)           :: XYslice_height
+   integer(IntKi), intent(in)           :: unit           !< Error status of the operation
+   integer(IntKi), intent(out)          :: ErrStat        !< Error status of the operation
```

Comment:
Are `ErrStat` and `ErrMsg` initialized in this function? Maybe I missed it.

Reply:
They aren't explicitly initialized, but get set either by `OpenFOutFile` or by the check on the slice height.  For completeness I can add them.

I also just noticed that there is no upper bounds check on the height, which could lead to an invalid index on `iz`.  I'll fix that also.